socorro/cron/crontabber_app.py

```diff
@@ -177,6 +184,11 @@ def mark_success(self, job_list_string):
             self._remember_success(job_class, success_date=now, duration=0)
 
     def main(self):
+        version_json_path = os.path.join(ROOT, "version.json")
+        if os.path.exists(version_json_path):
+            with open(version_json_path, 'r') as fp:
+                self.config.logger.info('version.json: ' + fp.read().strip())
+
```

Comment:
I fiddled with the code a bit and I think my idea of putting this in `SocorroApp` is a better solution. It covers both crontabber and processor apps so we don't need to do this twice. It also prints out the version information before all the configuration information. That's pretty handy.

Can you move it there, please?

Reply:
@willkg Thanks for reviewing!
I moved the function. Could you check it, please?